packrat/init.R

```diff
@@ -186,7 +186,7 @@ local({
     ## an 'installed from source' version
 
     ## -- InstallAgent -- ##
-    installAgent <- "InstallAgent: packrat 0.7.0"
+    installAgent <- "InstallAgent: packrat 0.9.1-1"
```

Comment:
Zonder dat deze nieuwe versie ook in packrat.lock staat, lijkt me dit niet een correcte wijziging.